package/tests/test_integrations/test_sqlite_store.py

```diff
@@ -1,6 +1,6 @@
 # We need to disable pylint because of this issue -
 #  https://github.com/pylint-dev/pylint/issues/8138
-# pylint: disable=E1102
+# pylint: disable=E1102,useless-suppression
```

Comment:
I do not see any file changes here, can you please comment on why this was added ?